session/bootstraptest/bootstrap_upgrade_test.go

```diff
@@ -335,7 +335,7 @@ func TestUpgradeVersionForUpgradeHTTPOp(t *testing.T) {
 	seLatestV := session.CreateSessionAndSetID(t, store)
 	ver, err = session.GetBootstrapVersion(seLatestV)
 	require.NoError(t, err)
-	require.Equal(t, session.SupportUpgradeHTTPOpVer+1, ver)
+	require.Less(t, session.SupportUpgradeHTTPOpVer, ver)
```

Comment:
It would be better to replace it with `require.Equal(t, session.CurrentBootstrapVersion+1, ver)`